x/scheduler/keeper/keeper.go

```diff
@@ -21,80 +22,78 @@ import (
 )
 
 type Keeper struct {
-	cdc        codec.BinaryCodec
-	storeKey   storetypes.StoreKey
-	memKey     storetypes.StoreKey
-	paramstore paramtypes.Subspace
+	cdc      codec.BinaryCodec
+	storeKey store.KVStoreService
+	memKey   storetypes.StoreKey
 
 	account   types.AccountKeeper
 	EvmKeeper types.EvmKeeper
 
 	ider keeperutil.IDGenerator
 
-	Chains map[xchain.Type]xchain.Bridge
+	Chains    map[xchain.Type]xchain.Bridge
+	authority string
 }
 
 func NewKeeper(
 	cdc codec.BinaryCodec,
-	storeKey,
+	storeKey store.KVStoreService,
 	memKey storetypes.StoreKey,
-	ps paramtypes.Subspace,
 	account types.AccountKeeper,
 	evmKeeper types.EvmKeeper,
 	chains []xchain.Bridge,
+	authority string,
 ) *Keeper {
-	// set KeyTable if it has not already been set
-	if !ps.HasKeyTable() {
-		ps = ps.WithKeyTable(types.ParamKeyTable())
-	}
 
 	cm := slice.MustMakeMapKeys(chains, func(c xchain.Bridge) xchain.Type {
 		return c.XChainType()
 	})
 
 	k := &Keeper{
-		cdc:        cdc,
-		storeKey:   storeKey,
-		memKey:     memKey,
-		paramstore: ps,
-		account:    account,
-		EvmKeeper:  evmKeeper,
-		Chains:     cm,
+		cdc:       cdc,
+		storeKey:  storeKey,
+		memKey:    memKey,
+		account:   account,
+		EvmKeeper: evmKeeper,
+		Chains:    cm,
+		authority: authority,
 	}
 
 	k.ider = keeperutil.NewIDGenerator(k, nil)
 
 	return k
 }
 
-func (k Keeper) GetAccount(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI {
+func (k Keeper) GetAccount(ctx context.Context, addr sdk.AccAddress) authtypes.AccountI {
 	return k.account.GetAccount(ctx, addr)
 }
 
-func (k Keeper) Logger(ctx sdk.Context) log.Logger {
-	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
+func (k Keeper) Logger(ctx context.Context) log.Logger {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	return sdkCtx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
```

Comment:
Use `liblog` for logging.

Reply:
Everywhere, exclusively.  No other logging calls in production code.

Reply:
Did you mean to replace the usage of fmt here? It was there in the master branch already [here](https://github.com/palomachain/paloma/blob/72fcb671899c8eb0557e5342cd7547e9655e14c5/x/scheduler/keeper/keeper.go#L75)

Reply:
```suggestion
		return liblog.FromSDKLogger(ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName)))
```

Reply:
Updated and pushed